sdks/java/io/debezium/build.gradle

```diff
@@ -90,13 +135,8 @@ task integrationTest(type: Test, dependsOn: processTestResources) {
   }
 }
 
-configurations.all (Configuration it) -> {
-  resolutionStrategy {
-    // Force protobuf 3 because debezium is currently incompatible with protobuf 4.
-    // TODO - remove this and upgrade the version of debezium once a proto-4 compatible version is available
-    // https://github.com/apache/beam/pull/33526 does some of this, but was abandoned because it still doesn't
-    // work with protobuf 4.
-    force "com.google.protobuf:protobuf-java:3.25.5"
-    force "com.google.protobuf:protobuf-java-util:3.25.5"
-  }
+project.tasks.each {
```

Comment:
I mean also remove these project.tasks.each { ... } (as well as in expansion-service build.gradle)

This has already been handled in https://github.com/apache/beam/blob/de76cfa16d36e2cb8dd1a1092710f0cd477e0f8d/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L1856

Reply:
Thank you, done!